tests/tripal/api/TripalJobsApiTest.php

```diff
@@ -13,8 +14,49 @@ class TripalJobsApiTest extends TripalTestCase {
    *
    * @test
    */
-  public function should_create_a_tripal_job() {
-    $job_id = tripal_add_job('Test adding jobs', 'tripal', 'tripal_tripal_cron_notification', [], 1);
+  public function testCreatingAJobWorks() {
+    $job_id = tripal_add_job('Test adding jobs', 'tripal',
+      'tripal_tripal_cron_notification', [], 1);
     $this->assertTrue(is_numeric($job_id));
   }
+
+  /** @test */
+  public function testRetrievingAJob() {
+    $job = factory('tripal_jobs')->create();
+
+    $job2 = tripal_get_job($job->job_id);
+
+    $this->assertNotEmpty($job2);
+    $this->assertObjectHasAttribute('job_id', $job2);
```

Comment:
Maybe add `$this->assertEquals($job2->job_id, $job->job_id)` as an easy test that it retrieved the right job?

Reply:
Added. Thanks for the suggestion!